sysfs/class_drm_amdgpu.go

```
@@ -117,6 +135,10 @@ func parseClassDRMAMDGPUCard(card string) (ClassDRMCardAMDGPUStats, error) {
 	if v, err := readDRMCardField(card, "unique_id"); err == nil {
 		stats.UniqueID = v
 	}
+	if n, t, err := readDevInfo(card); err == nil {
```

Should we silently fail here?

---

You mean something like this in case we couldn't retrieve the name & type?

```go
if err != nil {
    return ClassDRMCardAMDGPUStats{}, err
}
```

---

Yes